src/LDLFactorizations.jl

```diff
@@ -329,6 +329,83 @@ function ldl_solve!(n, B::AbstractMatrix{T}, Lp, Li, Lx, D, P) where T
   return B
 end
 
+# compute L*D*L'*x where x is a vector
+function ldl_ltmul!(n, x::AbstractVector, Lp, Li, Lx)
+    @inbounds for j = 1:n
+        xj = x[j]
+        @inbounds for p = Lp[j] : (Lp[j+1] - 1)
+            xj += Lx[p] * x[Li[p]]
+        end
+        x[j] = xj
+    end
+    return x
+end
+
+function ldl_dmul!(n, x::AbstractVector, D)
+    @inbounds for j = 1:n
+        x[j] *= D[j]
+    end
+    return x
+end
+
+function ldl_lmul!(n, x::AbstractVector, Lp, Li, Lx)
+    @inbounds for j = n:-1:1
+        xj = x[j]
+        @inbounds for p = Lp[j] : (Lp[j+1] - 1)
+            x[Li[p]] += Lx[p] * xj
+        end
+    end
+    return x
+end
+
+function ldl_mul!(n, x::AbstractVector, Lp, Li, Lx, D, P)
+    @views y = x[P]
+    ldl_ltmul!(n, y, Lp, Li, Lx)
+    ldl_dmul!(n, y, D)
+    ldl_lmul!(n, y, Lp, Li, Lx)
+  return x
+end
+
+# compute L*D*L'*X where X is a matrix
+function ldl_ltmul!(n, X::AbstractMatrix{T}, Lp, Li, Lx) where T
```

Comment:
Is it necessary to parametrize here?

Reply:
No it is not. Should I also remove parametrization for ldl_solve!() (lign 324) and for ldl_lsolve!(), ldl_dsolve!() and ldl_ltsolve!() ?

Reply:
Sure, but maybe in a separate PR. Thank you!